app/assets/javascripts/calendar/create_event.js

```diff
@@ -1,6 +1,7 @@
 $(function () {
     $(document).on('ajax:success', '#create-event-form', function () {
         $('#create-event-modal').modal('hide');
+        window.location.reload();
```

Comment:
Once we choose some cutting-edge shiny js framework, we should think about some better UX here ;)